packages/thumbprint-react/components/UIAction/plain.jsx

```diff
@@ -9,17 +9,28 @@ import styles from './plain.module.scss';
 const Plain = React.forwardRef((props, ref) => {
     warning(
         props.children || props.accessibilityLabel || (props.iconLeft && props.children),
-        'The prop `accessibilityLabel` must be provided to the button or link if `iconLeft` is provided but `children` is not. This helps users on screen readers navigate our content.',
+        'The prop `accessibilityLabel` must be provided to the button or link if `iconLeft` or `iconRight` are provided but `children` is not. This helps users on screen readers navigate our content.',
     );
 
-    const children = props.iconLeft ? (
-        <span className={styles.flexCenter}>
-            {props.iconLeft}
-            {props.children && <span className={styles.textContainer}>{props.children}</span>}
-        </span>
-    ) : (
-        props.children
-    );
+    let { children } = props;
+
+    if (props.iconLeft) {
+        children = (
+            <span className={styles.flexCenter}>
+                {props.iconLeft}
+                {props.children && <span className={styles.textContainer}>{props.children}</span>}
+            </span>
+        );
+    }
+
+    if (props.iconRight) {
```

Comment:
Should `iconLeft` and `iconRight` be mutually exclusive props? If so, a note about that should probably be added to the docs for the components